modular_skyrat/modules/nanotrasen_rep/code/nanotrasen_consultant.dm

```diff
@@ -58,7 +58,10 @@
 	suit = /obj/item/clothing/suit/armor/vest/nanotrasen_consultant
 	shoes = /obj/item/clothing/shoes/jackboots
 	head = /obj/item/clothing/head/nanotrasen_consultant
-	backpack_contents = list(/obj/item/melee/baton/telescopic)
+	backpack_contents = list(
+		/obj/item/melee/baton/telescopic = 1,
+		/obj/item/storage/box/gunset/nanotrasen_consultant = 1,
```

Comment:
the NT rep does not need a gun

Reply:
They've had the SC-2 since we got rid of ballistics and now you want to remove their gun now that we're moving to all crew being able to have a weapon?